### PR TITLE
Clarifying `autoUpdateWebhooks` behavior on docs

### PR DESCRIPTION
**Pull request by tchellomello**
## Proposed Changes
Enhances documentation by explaining the expected behavior when `autoUpdateWebhooks=false`

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [x] I have inspected the website preview for accuracy.
- [x] I have signed off my issue.

### CODE REVIEW

**Review by realshuting — APPROVED** [review]
@tchellomello - thanks for the clarification, it looks good!

File: content/en/docs/Installation/_index.md [hunk 1]
```diff
@@ -459,7 +459,7 @@ The following flags can also be used to control the advanced behavior of Kyverno
 7. `backgroundScan`: the interval (like 30s, 15m, 12h) for background processing. Default is set to 1h.
 8. `imagePullSecrets`: specifies secret resource names for image registry access credentials.
 9. `allowInsecureRegistry`: Allows Kyverno to work with insecure registries (i.e., bypassing certificate checks) either with [verifyImages](/docs/writing-policies/verify-images/) rules or [variables from image registries](/docs/writing-policies/external-data-sources/#variables-from-image-registries). Only for testing purposes. Not to be used in production situations.
-10. `autoUpdateWebhooks`: Set this flag to 'false' to disable auto-configuration of the webhook. Default is set to 'true'.
+10. `autoUpdateWebhooks`: Set this flag to 'false' to disable auto-configuration of the webhook. With this feature disabled, Kyverno creates a default webhook configuration (which match all kinds of resources), therefore, webhooks configuration via the configmap will be ignored. However, the user still can modify it by patching the webhook resource manually. Manual  Default is set to 'true'.
```
Comment (realshuting) [inline content/en/docs/Installation/_index.md:462]: Typo?
>..webhook resource manually. ~~Manual~~
```suggestion
10. `autoUpdateWebhooks`: Set this flag to 'false' to disable auto-configuration of the webhook. With this feature disabled, Kyverno creates a default webhook configuration (which match all kinds of resources), therefore, webhooks configuration via the configmap will be ignored. However, the user still can modify it by patching the webhook resource manually. Default is set to 'true'.
```

Comment (tchellomello) [reply content/en/docs/Installation/_index.md]: > Typo?
> 
> > ..webhook resource manually. ~Manual~

yes, let me fix it

Comment (realshuting) [reply content/en/docs/Installation/_index.md]: I have fixed it :)

**Pull request merged**